ui/raidboss/data/06-ew/raid/p3s.ts

```diff
@@ -375,6 +383,35 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P3S Death\'s Toll Number',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: ['ACA'], capture: true }),
+      // Force this to only run once without Conditions.targetIsYou()
+      // in case user is dead but needs to place fledgling flight properly
+      preRun: (data) => data.deathsToll = true,
+      // Delay callout until Ashen Eye start's casting
+      delaySeconds: 15.5,
+      infoText: (data, matches, output) => {
+        if (matches.target === data.me && !data.deathsTollPending) {
+          data.deathsTollPending = true;
+          return {
+            '01': output.outCardinals!(),
+            '02': output.outIntercards!(),
+            '04': output.middle!(),
+          }[matches.count];
+        }
+      },
+      outputStrings: {
+        middle: Outputs.middle,
+        outIntercards: {
+          en: 'Out + Intercards',
```

Comment:
Suggesting to put intercards first, because otherwise "Out + Intercards" and "Out + Cardinals" read very similarly.

```suggestion
          en: 'Intercards + Out',
```

Alternatively, I've also seen people do cardinals and inside (but not middle) for 2s, so I wonder a little bit if this should just call 1, 2, 4 and folks can use the UI to set a more specific string if they want? What do you think.

Reply:
I never thought about using the UI, namely because I have not used it for the most part. On one hand just numbers makes it easier for a person to translate to something else.